includes/Classifai/Watson/Linker.php

```diff
@@ -33,24 +33,36 @@ class Linker {
 	 * @param int   $post_id The post to link to.
 	 * @param array $output  The classification results from Watson NLU.
 	 * @param array $options Unused.
-
+	 *
 	 * @return void
 	 */
 	public function link( $post_id, $output, $options = [] ) {
+		$all_terms = [];
+
 		if ( ! empty( $output['categories'] ) ) {
-			$this->link_categories( $post_id, $output['categories'] );
+			$terms = $this->link_categories( $post_id, $output['categories'], true );
```

Comment:
Getting a PHPCS warning here because the equals signs aren't aligned. Similar issue on lines 48, 53 and 58